jobs/aws/eks-distro/main-1-18-postsubmits.yaml

```diff
@@ -35,11 +36,14 @@ postsubmits:
       containers:
       - name: build-container
         image: 316434458148.dkr.ecr.us-west-2.amazonaws.com/eks-distro/builder:afa151c76fdefe7248c1a75bb1a764c7e28fb334
+        env:
+        - name: TEST_ROLE_NAME
+          value: "TestBuildRole"
```

Comment:
If we are storing the role name here, don't we want the account id also? Or should we instead just store the role arn instead?